src/App.js

```diff
@@ -1,10 +1,15 @@
 import React from 'react';
 import './App.css';
 import { Home, About, Contact, Navigation } from './components';
+import { Route } from 'react-router-dom';
 
 const App = () => (
   <div>
     <Navigation />
+
+    <Route exact path='/' component={Home} />
+    <Route path='/about' component={About} />
```

Comment:
Try to use `exact` on all of them, and play around with it to see how it actually works and what it does. Personally when i wanna render a component on it's own page, i use `exact`